indexers/social_feed/social_feed.js

```diff
@@ -241,6 +243,82 @@ async function getBlock(block: Block) {
     }
   }
 
+    async function handlePromotion(
+    accountId,
+    blockHeight,
+    blockTimestamp,
+    receiptId,
+    promoteString
+  ) {
+    const promotion = JSON.parse(promoteString);
```

Comment:
This function could use a try/catch so that it's the same as the other handlers. Specifically, someone will eventually write some invalid data to their profile and break that JSON.parse.

Reply:
will do. I refactored the function into two functions handlePromotion and handleAddPromotion and in doing so brought the try catch into the latter without updating the former.